PeakCoreData/Core/Protocols/ManagedObjectType.swift

```diff
@@ -158,16 +158,14 @@ public extension ManagedObjectType {
 
 public extension ManagedObjectType where Self: UniqueIdentifiable {
     
-    typealias UniqueKeyValueType = Any
```

Comment:
Why not change this alias to AnyHashable instead of removing it?

Reply:
I've put this back